src/main/java/com/shipmentEvents/demo/EventHandler.java

```diff
@@ -67,7 +67,7 @@ private void processShipmentUpdates(final LambdaLogger logger) throws Interrupte
             filesToDelete.put(bucketName, filesProcessed);
         }
         final AmazonS3 s3Client = EventHandler.getS3Client();
-
+        boolean flag=fallse
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete LocalVariableDeclarationStatement